lib/ret/hub.ex

```diff
@@ -862,6 +862,14 @@ defimpl Canada.Can, for: Ret.Account do
     false
   end
 
+  def can?(
+        %Ret.Account{is_admin: true} = admin_account,
+        :delete_account,
+        %Ret.Account{is_admin: false} = account_to_delete
```

Comment:
Intent here is that you should not be able to delete admin accounts? I guess having to de-admin an account before deleting it adds a nice layer of added prevention from accidental deletion.

Reply:
Bingo